test/runtime/samples/window-event/main.svelte

```diff
@@ -3,6 +3,6 @@
 	export let height;
 </script>
 
-<svelte:window on:resize='{() => width = this.innerWidth, height = this.innerHeight}'/>
+<svelte:window on:resize={function () { width = this.innerWidth, height = this.innerHeight; }}/>
```

Comment:
i'm not sure what's the expected `this`, since it's an arrow function, so, it depends on where the arrow function is declared. and we could have hoisted it to module level, or it's within the `SvelteComponent`.

feel like we should warn about this?

Reply:
Svelte used to convert arrow functions used as event handlers into regular functions when compiling, which was actually a subtle bug, and this code was inadvertently depending on that bug. This was fixed a while ago, but we didn't notice the problem here because the test was disabled. Turning this into a regular function seems fine.